easybuild/easyconfigs/d/DAS_Tool/DAS_Tool-1.1.1-foss-2021a-R-4.1.0.eb

```diff
@@ -0,0 +1,59 @@
+easyblock = 'Tarball'
+
+name = 'DAS_Tool'
+version = '1.1.1'
```

Comment:
@smoretti Latest is `1.1.5`, let's update to that?

Reply:
Weird think for that one. It has a small patch file but it should be fine.

The problem is that it downloads a second time the source for R, and the second URL does not exist for 1.1.5 !!!

Reply:
Do you have more details on that?

Reply:
It does not find the second download link
`ERROR EasyBuild crashed with an error (at easybuild/base/exceptions.py:124 in __init__): Couldn't find file DASTool_1.1.5.tar.gz anywhere, and downloading it didn't work either... `